src/GrpcProtos/capture.proto

```diff
@@ -200,6 +200,7 @@ message ApiScopeStopAsync {
   int32 pid = 1;
```

Comment:
So as discussed a couple of `bytes` should be `repeated fixed64` for arena reasons.

Reply:
Done.